commands/service_deploy.go

```diff
@@ -53,8 +53,8 @@ func (c *serviceDeployCmd) runE(cmd *cobra.Command, args []string) error {
 			errors.New("To get more information, run: mesg-core service validate"),
 		}
 	}
-	fmt.Println("Service deployed with ID:", pretty.Success(id))
-	fmt.Printf("To start it, run the command: mesg-core service start %s\n", id)
+	fmt.Printf("%s Service deployed with ID: %v\n\t", pretty.SuccessSign, pretty.Success(id))
```

Comment:
it this `\t` wanted ? I feel it weird a bit to have everything shifted. The command why not but the message seems weird

<img width="516" alt="screen shot 2018-09-23 at 11 55 08 am" src="https://user-images.githubusercontent.com/1783126/45924332-9d63e700-bf27-11e8-86af-af31ed4340f1.png">

Reply:
I was talking about putting tab in front on `mesg-core...` command. `To start it` should be shifted to left

Reply:
I agree too, the tab should be only before `mesg-core...` i'm fixing it.